### ISSUE TITLE
False positive - :misplaced-docstring in clojure.test/deftest

### ISSUE THREAD

**Issue by NoahTheDuke** [clj-kondo/clj-kondo#2001]
**version**
```
$ clj-kondo --version
clj-kondo v2023.02.17
```

**platform**
Native clj-kondo on Linux, Ubuntu 22.10

**problem**
Placing a string as the second form of a `clojure.test/deftest` raises a `:misplaced-docstring` warning.

**repro**
```
$ echo '(ns example (:require [clojure.test :refer [deftest is]])) (deftest missing-docstring "this isnt a docstring" (is (= 1 1)))' | clj-kondo --lint -
<stdin>:1:87: warning: Misplaced docstring.
linting took 5ms, errors: 0, warnings: 1
```

**expected behavior**

I expect to see no warning, or maybe a warning about unused value? Either is fine.

I can make the PR.

**Comment by borkdude** [comment]
PR welcome!
